caravel/assets/javascripts/SqlLab/components/TabbedSqlEditors.jsx

```diff
@@ -1,14 +1,60 @@
 import React from 'react';
-import { DropdownButton, MenuItem, Tab, Tabs } from 'react-bootstrap';
+import { DropdownButton, MenuItem, Tab, Tabs, Popover, OverlayTrigger } from 'react-bootstrap';
 import { connect } from 'react-redux';
 import { bindActionCreators } from 'redux';
+import CopyToClipboard from '../../components/CopyToClipboard';
 import * as Actions from '../actions';
 import SqlEditor from './SqlEditor';
 import shortid from 'shortid';
 
 let queryCount = 1;
 
 class QueryEditors extends React.Component {
```

Comment:
this might be out of scope, but it would be great to fix this. classes should be the same name as their filename, so we should change this to `class TabbedSqlEditors extends...`